pkg/kubelet/network/cni/cni.go

```diff
@@ -189,6 +183,14 @@ func (plugin *cniNetworkPlugin) Name() string {
 	return CNIPluginName
 }
 
+func (plugin *cniNetworkPlugin) Status() error {
```

Comment:
This is called every 5 sec or so and is included when kubelet reports the node status. It's good to have consistent behavior for both kubenet and cniNetworkPlugin.